src/components/DataDecorator/Decorator/Decorator.stories.js

```diff
@@ -27,18 +27,38 @@ const storyProps = () => ({
     min: scoreThresholds[0],
     range: true,
   }),
-  href: text('Link (`href`)', href),
-  onClick: action('onClick'),
-  inert: boolean('Non-interactive (`inert`)', false),
   active: boolean('Active (`active`)', false),
   noIcon: boolean('No icon (`noIcon`)', false),
 });
 
 storiesOf(components('Decorator'), module)
-  .add('Default', () => <Decorator {...storyProps()} />)
+  .add('Default', () => (
+    <>
+      <p className="bx--type-body-long-01">This Decorator is inert.</p>
+      <p>
+        <Decorator {...storyProps()} />
+      </p>
+    </>
+  ))
   .add('Inline', () => (
     <p className="bx--type-body-long-01">
       This is an inline decorator <Decorator {...storyProps()} inline /> that
```

Comment:
- [ ] Can we ensure casing is consistent across the helper text for each of these stories? For example, either 'Decorator' or 'decorator'

Reply:
Sure thing 👍 But other than that, is this good to go?